platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/style/RuntimeStyleActivity.java

```diff
@@ -244,8 +244,8 @@ private void addParksLayer() {
                 fillAntialias(true)
         );
 
-        //Only show me parks
-        layer.setFilter(eq("type", "park"));
+        //Only show me parks (except westerpark with stroke-width == 3)
```

Comment:
@ivovandongen what do you mean with `except westerpark`, is it because we don't have that data in the layer, or is it because there's a filter to exclude that park somewhere else?